api/CredentialManagementAPI.json

```diff
@@ -0,0 +1,139 @@
+{
+  "api": {
+    "CredentialManagementAPI": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/Credential_Management_API",
+        "support": {
+          "webview_android": {
+            "version_added": "51"
+          },
+          "chrome": {
+            "version_added": "51"
+          },
+          "chrome_android": {
+            "version_added": "51"
+          },
+          "firefox": {
+            "version_added": null
+          },
+          "firefox_android": {
+            "version_added": null
+          },
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": "44"
+          },
+          "opera_android": {
+            "version_added": "44"
+          },
+          "safari": {
+            "version_added": null
+          },
+          "safari_ios": {
+            "version_added": null
+          }
+        },
+        "status": {
+          "experimental": true,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "subdomain": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Credential_Management_API#Subdomain-shared_credentials",
+          "description": "Subdomain-shared credentials",
+          "support": {
+            "webview_android": {
+              "version_added": "57"
+            },
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": "57"
+            },
+            "firefox": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": null
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "44"
+            },
+            "opera_android": {
+              "version_added": "44"
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "web_authentication": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Web_Authentication_API",
+          "description": "Web authentication",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "65",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "enable-webauthentication",
+                  "value_to_set": "true"
+                }
+              ]
```

Comment:
Chrome enabled this by default in 67. https://www.chromestatus.com/features/5669923372138496